### PR TITLE
Table primary2

### PR DESCRIPTION
**Pull request by Faryshta**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | yes |
| Breaks BC? | no (hope so) |
| Tests pass? | yes |
| Fixed issues | #11638 |

### CODE REVIEW

**Comment by Faryshta** [comment]
The `Schema::TYPE_PK` and similar constants are now deprecated.

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -11,7 +11,8 @@
 use yii\base\Object;
 
 /**
- * ColumnSchemaBuilder helps to define database schema types using a PHP interface.
+ * ColumnSchemaBuilder helps to define database schema types using a PHP
+ * interface.
```
Comment (cebe) [inline framework/db/ColumnSchemaBuilder.php]: ?

**Comment by cebe** [comment]
can you try to resubmit without all the automatic formatting changes? it is very hard to review this way.

**Comment by SilverFire** [comment]
The PR seems to be abandoned.

@Faryshta, @samdark

**Comment by samdark** [comment]
Won't be merged into 2.0.

**Pull request closed without merging**